sea/config.py

```diff
@@ -45,5 +47,23 @@ def get_namespace(self, namespace, lowercase=True, trim_namespace=True):
             rv[key] = v
         return ConstantsObject(rv)
 
+    def filter_env(self):
+        """
+        read environment variables and overwrite same name key's values.
+        `True` will be converted to python's bool `True`.
+        `1` will be converted to python's int `1`.
+        """
+        keys = self.keys()
+
+        for k in keys:
+            env_value = os.getenv(k)
+            if not env_value:
+                continue
+            value_type = type(self[k])
+            if value_type is bool:
+                self[k] = env_value == 'True'
```

Comment:
replace function with disutils.util.strtobol

Reply:
Package disutils has been marked as deprecated since 3.10 and will be removed from 3.12. I copied it to sea.utils.